Source/DafnyCore/DafnyOptions.cs

```diff
@@ -1284,6 +1284,8 @@ verification outcome.
         be used if the program already contains a main method. Note that
         /compile:3 or 4 must be provided as well to actually execute
         this main method!
+        Beware: for now, the `Main` method generated by `/runAllTests` always
```

Comment:
```suggestion
        WARNING: for now, the `Main` method generated by `/runAllTests` always
```

Just turning up the volume a little :)